scripts/wait-for-it/LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
Hi @AlexanderBarbie, could you please move this license into our dedicated [THIRD-PARTY-NOTICES file](https://github.com/corona-warn-app/cwa-server/blob/master/THIRD-PARTY-NOTICES)? Thanks.